filters-share/specific_URL.txt

```diff
@@ -354,7 +354,6 @@
 ||hellodd.com/adManager/css/adManager.css
 ||hellodd.com/bannerpop/uploads/
 ||hellot.net/data/banner/
-||dailysecu.com/bannerpop/uploads/
```

Comment:
This blocks advertisement on https://dailysecu.com/.

Reply:
This entry appears twice: https://github.com/List-KR/List-KR/blob/master/filters-share/specific_URL.txt#L235 and https://github.com/List-KR/List-KR/blob/master/filters-share/specific_URL.txt#L358